mod_import_anymeta.erl

```diff
@@ -434,7 +437,10 @@ import_thing(Host, AnymetaId, Thing, Stats, Context) ->
                         {anymeta_rsc_uri, RscUri},
                         {anymeta_host, Host},
                         {language, Langs},
-                        {is_authoritative, Authoritative}
+                        {is_authoritative, Authoritative},
+                        {rights, Rights},
+                        {findable, Findable},
+                        {matchable, Matchable}
```

Comment:
Maybe better to make these `is_findable` and `is_matchable` ? Then they adhere to the standard to start boolean properties with `is_...`